.github/workflows/ci-pip-tools.yml

```diff
@@ -36,6 +36,10 @@ jobs:
         working-directory: my_project
         run: pip install .[dev]
```

Comment:
I think that installing this in editable mode should solve the issues. True, it is not needed, but it causes no harm and might be useful to keep mypy happy.